src/read_write/raw.rs

```diff
@@ -226,12 +226,19 @@ impl RawNodeReader {
                             .attributes
                             .insert(key.to_owned(), AttributeData::U8(attr));
                     }
-                    AttributeDataType::I64 => {
+                    AttributeDataType::U16 => {
```

Comment:
How about defining a macro for all the different types here, too?

Reply:
I thought about it, but I don't imagine it would help readability since there are many slightly different cases: U8 vs I8 vs (U16, U32, U64, I16, I32, I64) vs U8Vec3 vs F64Vec3.

Reply:
Hm, how about a `ReadLE` trait, which covers these cases (similar to what we do in `WriteLE`) and then the macro could just use `read_le` in combination with `batch.attributes.insert(...)`?

Reply:
Thinking about it, how about doing that in a follow-up PR that also (hopefully) improves the module structure? IMO it would make sense to split off a `little_endian` module that can read/write our data types and `Vec`s of them from/to arbitrary `Read` and `Write` instances.

Reply:
Sounds good, can you please add a TODO here and open a ticket for it?